repos/rust-lang/annotate-snippets-rs.toml

```
@@ -0,0 +1,15 @@
+org = "rust-lang"
+name = "annotate-snippets-rs"
+description = "Library for snippet annotations"
+bots = []
+
+[access.teams]
+compiler = "write"
+devtools = "write"
```

[I proposed](https://rust-lang.zulipchat.com/#narrow/stream/147480-t-compiler.2Fwg-diagnostics/topic/assigning.20team.20ownership.20of.20annotate-snippets) `wg-diagnostics` and `t-cargo` (for now) being the owners of this repository instead of `t-compiler` and `t-devtools` and had positive responses, so I think we should go with that.
```suggestion
cargo = "write"
wg-diagnostics = "write"
```